### PR TITLE
[FLINK-13037][docs-zh] Translate "Concepts -> Glossary" page into Chinese

### PR DESCRIPTION
**Pull request by yangjf2019**
## What is the purpose of the change

*Translate "Concepts -> Glossary" page into Chinese.*

## Brief change log

  - *I modified the glossary.zh.md file .*

## Verifying this change

*I have  verified the changes by starting the build script in preview mode.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no )
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no )
  - The serializers: ( no )
  - The runtime per-record code paths (performance sensitive): ( no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: ( no )
  - The S3 file system connector: ( no )

## Documentation

  - Does this pull request introduce a new feature? (  no )
  - If yes, how is the feature documented? ( not documented )

Hi, @wuchong  @TisonKun  ,I replace a new account, so open a new PR, but the content of file is not change, here is the old [PR](https://github.com/apache/flink/pull/9414). Please Take A Look .

### CODE REVIEW

File: docs/concepts/glossary.zh.md [hunk 1]
```diff
@@ -25,142 +25,92 @@ under the License.
...
-usually executed by a [Function](#function). Sources and Sinks are special Operators for data
-ingestion and data egress.
+[Logical Graph](#logical-graph) 的节点。算子执行某种操作，该操作通常由 [Function](#function) 执行。Source 和 Sink 是数据输入和数据输出的特殊算子。
 
 #### Operator Chain
 
-An Operator Chain consists of two or more consecutive [Operators](#operator) without any
-repartitioning in between. Operators within the same Operator Chain forward records to each other
-directly without going through serialization or Flink's network stack.
+算子链由两个或多个连续的 [Operator](#operator) 组成，两者之间没有任何的重新分区。同一算子链中的算子可以直接相互记录，而无需通过序列化或 Flink 的网络堆栈。
```
Comment (xintongsong) [inline docs/concepts/glossary.zh.md]: “直接相互记录” -> “相互间直接传递record”

File: docs/concepts/glossary.zh.md [hunk 2]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### Partition
 
-A partition is an independent subset of the overall data stream or data set. A data stream or
-data set is divided into partitions by assigning each [record](#Record) to one or more partitions.
-Partitions of data streams or data sets are consumed by [Tasks](#task) during runtime. A
-transformation which changes the way a data stream or data set is partitioned is often called
-repartitioning.
+分区是整个数据流或数据集的独立子集。通过将每个 [Record](#record) 分配给一个或多个分区，来把数据流或数据集划分为多个分区。在运行期间，[Task](#task) 会使用数据流或数据集的分区。改变数据流或数据集分区方式的转换通常称为重分区。
```
Comment (xintongsong) [inline docs/concepts/glossary.zh.md]: 使用 -> 消费

File: docs/concepts/glossary.zh.md [hunk 3]
```diff
@@ -25,142 +25,92 @@ under the License.
...
-[Functions](#Function) receive records as input and emit records as output.
+Record 是数据集或数据流的组成元素。[Operator](#operator) 和 [Function](#Function)接收 record 作为输入，并将 record 作为输出发出。
 
 #### Flink Session Cluster
 
-A long-running [Flink Cluster](#flink-cluster) which accepts multiple [Flink Jobs](#flink-job) for
-execution. The lifetime of this Flink Cluster is not bound to the lifetime of any Flink Job.
-Formerly, a Flink Session Cluster was also known as a Flink Cluster in *session mode*. Compare to
-[Flink Application Cluster](#flink-application-cluster).
+长时间运行的 [Flink Cluster](#flink-cluster)，它可以接受多个 [Flink Job](#flink-job) 的执行。此 [Flink Cluster](#flink-cluster) 的生命周期不受任何 [Flink Job](#flink-job) 生命周期的约束限制。以前，Flink Session Cluster 在 *session mode* 中也称为 [Flink Cluster](#flink-cluster)，和 [Flink Application Cluster](#flink-application-cluster) 作对比。
```
Comment (xintongsong) [inline docs/concepts/glossary.zh.md]: Flink Session Cluster 也称为 session mode 的 Flink Cluster

File: docs/concepts/glossary.zh.md [hunk 4]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### State Backend
 
-For stream processing programs, the State Backend of a [Flink Job](#flink-job) determines how its
-[state](#managed-state) is stored on each TaskManager (Java Heap of TaskManager or (embedded)
-RocksDB) as well as where it is written upon a checkpoint (Java Heap of
-[Flink Master](#flink-master) or Filesystem).
+对于流处理程序，[Flink Job](#flink-job) 的 State Backend 决定了其 [state](#managed-state) 是如何存储在每个 TaskManager（ TaskManager 或（嵌入式）RocksDB 的 Java 堆）上，以及它在 checkpoint 上的写入位置（ [Flink Master](#flink-master) 或 Filesystem 的 Java 堆）。
```
Comment (xintongsong) [inline docs/concepts/glossary.zh.md]: “TaskManager 或（嵌入式）RocksDB 的 Java 堆” -> "TaskManager 的 Java 堆或（嵌入式） RocksDB"
"它在 checkpoint 上的写入位置" -> "它在 checkpoint 时的写出位置"
“Flink Master 的 Java 堆或 Filesystem”

File: docs/concepts/glossary.zh.md [hunk 5]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### Sub-Task
 
-A Sub-Task is a [Task](#task) responsible for processing a [partition](#partition) of
-the data stream. The term "Sub-Task" emphasizes that there are multiple parallel Tasks for the same
-[Operator](#operator) or [Operator Chain](#operator-chain).
+Sub-Task 是负责处理数据流 [Partition](#partition) 的 [Task](#task)。"Sub-Task"是一个强调相同的 [Operator](#operator) 或者 [Operator Chain](#operator-chain) 具有多个并行 Task 的术语。
```
Comment (xintongsong) [inline docs/concepts/glossary.zh.md]: 这里需要强调 Sub-Task 是处理 Data Stream **其中一个** Partition 的

File: docs/concepts/glossary.zh.md [hunk 6]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### Transformation
 
-A Transformation is applied on one or more data streams or data sets and results in one or more
-output data streams or data sets. A transformation might change a data stream or data set on a
-per-record basis, but might also only change its partitioning or perform an aggregation. While
-[Operators](#operator) and [Functions](#function)) are the "physical" parts of Flink's API,
-Transformations are only an API concept. Specifically, most - but not all - transformations are
-implemented by certain [Operators](#operator).
+Transformation 应用于一个或多个数据流或数据集，并产生一个或多个输出数据流或数据集。Transformation 可能会在每个记录的基础上更改数据流或数据集，但也可以只更改其分区或执行聚合。虽然 [Operator](#operator) 和 [Function](#function) 是 Flink API 的"物理"部分，但 Transformation 只是一个 API 概念。具体来说，大多数（但不是全部）转换是由某些 [Operator](#operator) 实现的。
```
Comment (xintongsong) [inline docs/concepts/glossary.zh.md]: “大多数（但不是全部）转换” 这里最好也用transformation和前文保持一致

**Comment by yangjf2019** [comment]
Sure , @xintongsong Thanks for your review. Not all translate are accurate, I accept your some comments, and  update this commit.

File: docs/concepts/glossary.zh.md (hunk 4 above)
Comment (yangjf2019) [reply docs/concepts/glossary.zh.md]: 这里表达的意思应该就是是“写入”，而不是“写出”。

File: docs/concepts/glossary.zh.md [hunk 7]
```diff
@@ -25,142 +25,92 @@ under the License.
...
+Function 是由用户实现的，并封装了 Flink 程序的应用程序逻辑。大多数 Function 都由相应的 [Operator](#operator) 封装。
 
 #### Instance
 
-The term *instance* is used to describe a specific instance of a specific type (usually
-[Operator](#operator) or [Function](#function)) during runtime. As Apache Flink is mostly written in
-Java, this corresponds to the definition of *Instance* or *Object* in Java. In the context of Apache
-Flink, the term *parallel instance* is also frequently used to emphasize that multiple instances of
-the same [Operator](#operator) or [Function](#function) type are running in parallel.
+Instance 常用于描述运行时的特定类型(通常是 [Operator](#operator) 或者 [Function](#function))的一个具体实例。由于 Apache Flink 主要是用 Java 编写的，所以，这与 Java 中的 Instance 或 Object 的定义相对应。在 Apache Flink 的上下文中，*parallel instance* 也常用于强调同一 [Operator](#operator) 或者 [Function](#function) 的多个 instance 以并行的方式运行。
```
Comment (TisonKun) [inline docs/concepts/glossary.zh.md]:
```suggestion
Instance 常用于描述运行时的特定类型(通常是 [Operator](#operator) 或者 [Function](#function))的一个具体实例。由于 Apache Flink 主要是用 Java 编写的，所以，这与 Java 中的 *Instance* 或 *Object* 的定义相对应。在 Apache Flink 的上下文中，*parallel instance* 也常用于强调同一 [Operator](#operator) 或者 [Function](#function) 的多个 instance 以并行的方式运行。
```

File: docs/concepts/glossary.zh.md [hunk 8]
```diff
@@ -25,142 +25,92 @@ under the License.
...
-A logical graph is a directed graph describing the high-level logic of a stream processing program.
-The nodes are [Operators](#operator) and the edges indicate input/output-relationships or
-data streams or data sets.
+Logical Graph 是一种描述流处理程序的高阶逻辑有向图。节点是[Operator](#operator)，边代表输入/输出关系、数据流和数据集中的之一。
 
 #### Managed State
 
-Managed State describes application state which has been registered with the framework. For
-Managed State, Apache Flink will take care about persistence and rescaling among other things.
+Managed State 描述了已在框架中注册的应用程序的托管状态。对于托管状态，Apache Flink 会负责持久化和重伸缩等事=事宜。
```
Comment (TisonKun) [inline docs/concepts/glossary.zh.md]:
```suggestion
Managed State 描述了已在框架中注册的应用程序的托管状态。对于托管状态，Apache Flink 会负责持久化和重伸缩等事宜。
```

File: docs/concepts/glossary.zh.md [hunk 9]
```diff
@@ -25,142 +25,92 @@ under the License.
...
-usually executed by a [Function](#function). Sources and Sinks are special Operators for data
-ingestion and data egress.
+[Logical Graph](#logical-graph) 的节点。算子执行某种操作，该操作通常由 [Function](#function) 执行。Source 和 Sink 是数据输入和数据输出的特殊算子。
 
 #### Operator Chain
 
-An Operator Chain consists of two or more consecutive [Operators](#operator) without any
-repartitioning in between. Operators within the same Operator Chain forward records to each other
-directly without going through serialization or Flink's network stack.
+算子链由两个或多个连续的 [Operator](#operator) 组成，两者之间没有任何的重新分区。同一算子链内的算子可以彼此直接传递 record，而无需通过序列化或 Flink 的网络堆栈。
```
Comment (TisonKun) [inline docs/concepts/glossary.zh.md]:
```suggestion
算子链由两个或多个连续的 [Operator](#operator) 组成，两者之间没有任何的重新分区。同一算子链内的算子可以彼此直接传递 record，而无需通过序列化或 Flink 的网络栈。
```

@wuchong I'm not sure whether we have a consensus here but I always tend to use term "堆栈"

File: docs/concepts/glossary.zh.md [hunk 10]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### Transformation
 
-A Transformation is applied on one or more data streams or data sets and results in one or more
-output data streams or data sets. A transformation might change a data stream or data set on a
-per-record basis, but might also only change its partitioning or perform an aggregation. While
-[Operators](#operator) and [Functions](#function)) are the "physical" parts of Flink's API,
-Transformations are only an API concept. Specifically, most - but not all - transformations are
-implemented by certain [Operators](#operator).
+Transformation 应用于一个或多个数据流或数据集，并产生一个或多个输出数据流或数据集。Transformation 可能会在每个记录的基础上更改数据流或数据集，但也可以只更改其分区或执行聚合。虽然 [Operator](#operator) 和 [Function](#function) 是 Flink API 的"物理"部分，但 Transformation 只是一个 API 概念。具体来说，大多数（但不是全部）Transformation 是由某些 [Operator](#operator) 实现的。
```
Comment (TisonKun) [inline docs/concepts/glossary.zh.md]:
```suggestion
Transformation 应用于一个或多个数据流或数据集，并产生一个或多个输出数据流或数据集。Transformation 可能会在每个记录的基础上更改数据流或数据集，但也可以只更改其分区或执行聚合。虽然 [Operator](#operator) 和 [Function](#function) 是 Flink API 的“物理”部分，但 Transformation 只是一个 API 概念。具体来说，大多数（但不是全部）Transformation 是由某些 [Operator](#operator) 实现的。
```

**Comment by yangjf2019** [comment]
@TisonKun ,Thanks for your helping ,I fixed them.

File: docs/concepts/glossary.zh.md [hunk 11]
```diff
@@ -25,142 +25,92 @@ under the License.
...
-[Functions](#Function) receive records as input and emit records as output.
+Record 是数据集或数据流的组成元素。[Operator](#operator) 和 [Function](#Function)接收 record 作为输入，并将 record 作为输出发出。
 
 #### Flink Session Cluster
 
-A long-running [Flink Cluster](#flink-cluster) which accepts multiple [Flink Jobs](#flink-job) for
-execution. The lifetime of this Flink Cluster is not bound to the lifetime of any Flink Job.
-Formerly, a Flink Session Cluster was also known as a Flink Cluster in *session mode*. Compare to
-[Flink Application Cluster](#flink-application-cluster).
+长时间运行的 [Flink Cluster](#flink-cluster)，它可以接受多个 [Flink Job](#flink-job) 的执行。此 [Flink Cluster](#flink-cluster) 的生命周期不受任何 [Flink Job](#flink-job) 生命周期的约束限制。以前，Flink Session Cluster 也称为 *session mode* 的 [Flink Cluster](#flink-cluster)，和 [Flink Application Cluster](#flink-application-cluster) 作对比。
```
Comment (wuchong) [inline docs/concepts/glossary.zh.md]:
```suggestion
长时间运行的 [Flink Cluster](#flink-cluster)，它可以接受多个 [Flink Job](#flink-job) 的执行。此 [Flink Cluster](#flink-cluster) 的生命周期不受任何 [Flink Job](#flink-job) 生命周期的约束限制。以前，Flink Session Cluster 也称为 *session mode* 的 [Flink Cluster](#flink-cluster)，和 [Flink Application Cluster](#flink-application-cluster) 相对应。
```

File: docs/concepts/glossary.zh.md [hunk 12]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### State Backend
 
-For stream processing programs, the State Backend of a [Flink Job](#flink-job) determines how its
-[state](#managed-state) is stored on each TaskManager (Java Heap of TaskManager or (embedded)
-RocksDB) as well as where it is written upon a checkpoint (Java Heap of
-[Flink Master](#flink-master) or Filesystem).
+对于流处理程序，[Flink Job](#flink-job) 的 State Backend 决定了其 [state](#managed-state) 是如何存储在每个 TaskManager 的 Java 堆栈（嵌入式）RocksDB 上，以及它在 checkpoint 时的写入位置（ [Flink Master](#flink-master) 的 Java 堆或者 Filesystem）。
```
Comment (wuchong) [inline docs/concepts/glossary.zh.md]:
```suggestion
对于流处理程序，[Flink Job](#flink-job) 的 State Backend 决定了其 [state](#managed-state) 是如何存储在每个 TaskManager 上的（ TaskManager 的 Java 堆栈或嵌入式 RocksDB），以及它在 checkpoint 时的写入位置（ [Flink Master](#flink-master) 的 Java 堆或者 Filesystem）。
```

**Comment by yangjf2019** [comment]
Hi @wuchong @TisonKun @xintongsong Thanks for your review and helping.

**Pull request closed without merging**